spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "./vcr"
```

Comment:
Has VCR been validated by the technical-design-authority? What about security!? Report to the five-factor room immediately